File: result/bubbleSort.py
a = [None] * 100
def main():
	global a
	num = None
	print("Input the array length: ")
	num = eval(input())
	print("Input integers: ")
	i = 0
	while (i < num):
		a[i] = eval(input())
		i = (i + 1)
	i = (num - 1)
	while (i > 0):
		j = 0
		while (j < i):
			if (a[j] > a[(j + 1)]):
				tmp = a[j]
				a[j] = a[(j + 1)]
				a[(j + 1)] = tmp
			j = (j + 1)
		i = (i - 1)
	print("Result: ")
	i = 0
	while (i < num):
		print("%d " % (a[i], ))
		i = (i + 1)
	return 0

if __name__ == '__main__':
	main()

File: exe/bubbleSort.c
int a[100];

int main()
{
    int num;
    printf("Input the array length: ");
    scanf("%d", &num);
    printf("Input integers: ");
    for (int i = 0; i < num; i = i + 1)
    {
        scanf("%d", &a[i]);
    }
    for (int i = num - 1; i > 0; i = i - 1)
    {
        for (int j = 0; j < i; j = j + 1)
        {
            if (a[j] > a[j + 1])
            {
                int tmp = a[j];
                a[j] = a[j + 1];
                a[j + 1] = tmp;
            }
        }
    }
    printf("Result: ");
    for (int i = 0; i < num; i = i + 1)
    {
        printf("%d ", a[i]);
    }

    return 0;
}